app/views/books/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= simple_form_for @book do |form| %>
+  <%= form.label :title, t('simple_form.labels.title') %>
+  <%= form.text_field :title %>
```

Comment:
For `simple_form` I think you can do `form.input` and it will do the label and the text field in one go